SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf

```diff
@@ -502,6 +517,14 @@ Consensus:Review.</skos:editorialNote>
 		<rdfs:label xml:lang="en">private label m b s deal</rdfs:label>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-dbt-ab-mbs;RealEstateMortgageInvestmentConduit">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-fund-fund;SpecialPurposeVehicle"/>
+		<rdfs:label xml:lang="en">real estate mortgage investment conduit</rdfs:label>
+		<skos:definition xml:lang="en">special purpose vehicle or debt instrument that pools mortgage loans together and issues mortgage-backed securities</skos:definition>
+		<fibo-fnd-utl-av:abbreviation xml:lang="en">REMIC</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">A real estate mortgage investment conduit may be organized as a partnership, a trust, a corporation, or an association and is exempt from federal taxes.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
make clear that this is the default type in US

Reply:
@rivettp We made some changes to this in FCT discussion to clarify.